### ISSUE TITLE
Add support for exporting JSON manifest into other formats

### ISSUE THREAD

**Issue by tduffield** [chef/license_scout#151]
Add a `license_scout export` command that takes in 1) a manifest file and 2) a format. We then will convert the JSON file into that format. 

## Example
```
license_scout export --file my-dependency-licenses.json --format csv
```

## Initial Support Formats
- [ ] Comma separated files (csv)  [Spec](http://www.creativyst.com/Doc/Articles/CSV/CSV01.htm#FileFormat)

**Comment by danielsdeleo** [comment]
Just wanna note that "CSV" is not a hard requirement, we just need something that imports into a spreadsheet easily.

**Comment by tduffield** [comment]
@danielsdeleo yep - but CSV I think it the easiest and most globally recognized way to do that
